app/data/generators/events.js

```diff
@@ -1,14 +1,19 @@
-module.exports = (faker, params) => {
+const DateHelper = require('../helpers/dates');
+const faker = require('faker')
+faker.locale = 'en_GB'
+
+module.exports = (params) => {
   const events = { items: [] }
+  const now = new Date(2020, 8, 12)
```

Comment:
I presume u will abstract this out another time? (please don't just resolve this)

Reply:
Yes. We'll create a helper file for faker defaults.